### PR TITLE
Swift 4 version

### PR DESCRIPTION
**Pull request by GabrielaBezerra**
I updated the code for Swift 4. I hope it's going to be useful to someone, just like the objective-c code was useful to me :)

### CODE REVIEW

**Review by nnagulavancha-incomm — APPROVED** [review]

File: obj-c/reCaptcha/ViewController.m [hunk 1]
```diff
@@ -48,7 +48,7 @@ - (void)initWebkitView {
                             configuration:wkConf];
 
     wk.backgroundColor = [UIColor clearColor];
-    wk.opaque = NO;
```
Comment (ThinkChaos) [inline obj-c/reCaptcha/ViewController.m]: Please undo this whitespace change

File: swift/reCaptcha/script.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+ (function() {
```
Comment (ThinkChaos) [inline swift/reCaptcha/script.js]: Please keep move this file up a directory and share it between the projects

File: swift/reCaptcha/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      print("wk is null")
+    }
+  }
+  
+  func captchaDidSolve(response: String) {
+    print("response: \(response)")
+  }
+  
+  func captchaDidExpire() {
+    print("Captcha Expired")
```
Comment (ThinkChaos) [inline swift/reCaptcha/ViewController.swift]: Either use a lowercase e in "expired", or revert the change in the Objective-C version

File: obj-c/reCaptcha.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -113,6 +113,7 @@
 				TargetAttributes = {
 					76B3CA2A1D74AE3700D20A7C = {
 						CreatedOnToolsVersion = 7.3.1;
+						DevelopmentTeam = 526YR6WHWC;
```
Comment (ThinkChaos) [inline obj-c/reCaptcha.xcodeproj/project.pbxproj]: Please remove your development team info from both projects

**Review by ThinkChaos — CHANGES_REQUESTED** [review]

**Comment by ThinkChaos** [comment]
Thanks for the PR!

Sorry I didn't respond earlier, I missed that you had moved the Objective-C version into a directory, and thought you had just removed it.  
I should've at least acknowledged this PR.

Now that I saw that's not the case, I'll happily merge this if you could just change the things I commented on.

**Pull request closed without merging**